scripts/src/chartprreview/chartprreview.py

```diff
@@ -413,17 +501,24 @@ def main():
         help="API URL for the pull request",
     )
     args = parser.parse_args()
+
     category, organization, chart, version = get_modified_charts(
         args.directory, args.api_url
     )
     verify_user(args.directory, args.username, category, organization, chart)
     check_owners_file_against_directory_structure(
-        args.directory, args.username, category, organization, chart
+        args.directory, category, organization, chart
     )
+
+    report_generated = os.environ.get("REPORT_GENERATED")
+    generated_report_path = os.environ.get("GENERATED_REPORT_PATH")
+    generated_report_info_path = os.environ.get("REPORT_SUMMARY_PATH")
+    env = Env()
+    web_catalog_only = env.bool("WEB_CATALOG_ONLY", False)
+
     submitted_report_path = os.path.join(
```

Comment:
Can you elaborate a bit on what's happening here and why? I see some things got moved up, but I also see some params are being changed? These are more than documentation/comment changes, so I want to be sure of the intention behind them. My guess is unused parameters.